### PR TITLE
Add Portuguese translation by Bruno Almeida and Laureano Macedo

### PR DESCRIPTION
**Pull request by brunoalmeida81**

### CODE REVIEW

File: resource/translations/skosmos_pt.po [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+#: /tmp/cache/587c83c13194c/0b/0bbfc29e05e244fa07889ce1678840902f16e5f76560e00137de0cc3bb084b8b.php:199
+msgid "helper_help"
+msgstr "Passe com o cursor por cima do texto sublinhado para ver informações sobre a propriedade."
+
+#: /tmp/cache/587c83c13194c/0b/0bbfc29e05e244fa07889ce1678840902f16e5f76560e00137de0cc3bb084b8b.php:201
+msgid "search_example_text"
+msgstr "Para pesquisa com truncatura, utilize o símbolo * como, p. ex., em *medieval ou *social*. A pesquisa utiliza automaticamente a truncatura em final de palavra, mesmo que o símbolo não seja introduzido: hist dará os mesmos resultados que hist*."
+
+msgid "feedback_enter_name_email"
+msgstr "Introduza o seu nome e correio eletrónico se deseja receber uma resposta direta. Também pode submeter os seus comentários anonimamente. Nota: os comentários sobre um vocabulários específico são referidos diretamente ao seu administrador."
```
Comment (kinow) [inline resource/translations/skosmos_pt.po:767]: sobre um vocabulários

sobre vocabulários (I think, for concordance)

File: resource/translations/skosmos_pt.po [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+msgid "Concept language"
+msgstr "Língua"
+
+#: /tmp/cache/587c83c13194c/ab/abd3710fda280028e428e4bc415383b12c9a30cc34eadffd21ff4130ae796b99.php:118
+msgid "Count"
+msgstr "Número"
+
+#: /tmp/cache/587c83c13194c/ab/abd3710fda280028e428e4bc415383b12c9a30cc34eadffd21ff4130ae796b99.php:157
+msgid "Download this vocabulary as SKOS/RDF:"
+msgstr "Descarregar este vocabulário em SKOS/RDF:"
```
Comment (kinow) [inline resource/translations/skosmos_pt.po:716]: Interesting! This one would be "como SKOS/RDF" for me (Brazilian).

File: resource/translations/skosmos_pt.po [hunk 3]
```diff
@@ -0,0 +1,770 @@
...
+msgid "isothes:narrowerPartitive"
+msgstr "Conceitos específicos (parte-todo)"
+
+#: /tmp/cache/587c83c09abaa/4e/4e05e5051b20c875609479922a558d59b8500ae386eaf79bc460e69d8781a753.php:95
+msgid "isothes:superGroup"
+msgstr "Supergrupo"
+
+#: /tmp/cache/587c83c09abaa/58/58e9c1cebb82d52daf56fab57cd7b86ed47b1d8a86a915177156746eef73248e.php:66
+msgid "layout designed by Hahmo"
+msgstr "layout por Hahmo Design"
```
Comment (kinow) [inline resource/translations/skosmos_pt.po:478]: layout criado por Hahmo?

**Comment by osma** [comment]
Thanks! It seems that you created the translations in Transifex, then exported them into files (perhaps using the fetch-from-transifex script) and created this PR, right? That's the correct way, just checking.

Two options for merging (perhaps addressing @kinow's comments above), what's your preference:
1. Merge this as-is, then if there are any further changes they should be done in Transifex first and then create subsequent PRs
2. Do the changes right now in Transifex and then update this PR before merging.

We're in the middle of a sprint week and plan to do a 2.6 release early next week. It would be best to merge this (either option above) by Friday so it gets into the release.

**Comment by osma** [comment]
For @kinow, if you want to create a Brazilian Portuguese translation, there is already a pt-BR language registered on Transifex, currently with ~8% of strings translated: https://www.transifex.com/national-library-of-finland/skosmos/language/pt_BR/

File: resource/translations/skosmos_pt.po (hunk 1 above)
Comment (brunoalmeida81) [reply resource/translations/skosmos_pt.po:767]: You are quite right. I will change it to "sobre vocabulários".

File: resource/translations/skosmos_pt.po (hunk 2 above)
Comment (brunoalmeida81) [reply resource/translations/skosmos_pt.po:716]: It seems that "descarregar como <format>" is more used, so I will make this change.

File: resource/translations/skosmos_pt.po (hunk 3 above)
Comment (brunoalmeida81) [reply resource/translations/skosmos_pt.po:478]: I agree. Correction made.

**Comment by brunoalmeida81** [comment]
> Thanks! It seems that you created the translations in Transifex, then exported them into files (perhaps using the fetch-from-transifex script) and created this PR, right? That's the correct way, just checking.
> 
> Two options for merging (perhaps addressing @kinow's comments above), what's your preference:
> 
>     1. Merge this as-is, then if there are any further changes they should be done in Transifex first and then create subsequent PRs
> 
>     2. Do the changes right now in Transifex and then update this PR before merging.
> 
> 
> We're in the middle of a sprint week and plan to do a 2.6 release early next week. It would be best to merge this (either option above) by Friday so it gets into the release.

Thank you! Yes, I completed the pt translation which someone else had already started in Transifex. I followed the instructions in your [wiki](https://github.com/NatLibFi/Skosmos/wiki/Translation) to export the translation to a .po file, and then ran trans_script to generate the .mo file.

I will do the changes suggested by @kinow in Transifex, and update the PR accordingly.

File: resource/translations/skosmos_pt.po (hunk 3 above)
Comment (kinow) [reply resource/translations/skosmos_pt.po:478]: Oh, sorry, I'm going through the translation process for Brazilian Portuguese, and noticed that in Transifex, it shows "layout designed by Hahmo Design", which would make sense "layout por Hahmo Design" @brunoalmeida81.

Comment (kinow) [reply resource/translations/skosmos_pt.po:478]: (I'm actually using the same one)

Comment (brunoalmeida81) [reply resource/translations/skosmos_pt.po]: No harm done. "layout criado por Hahmo Design" is fine.

**Comment by osma** [comment]
@brunoalmeida81 Ready for merging now?

**Comment by brunoalmeida81** [comment]
> @brunoalmeida81 Ready for merging now?

Ready.

**Comment by osma** [comment]
Thanks a lot, both of you!

**Pull request merged**